joomla/joomla-cms
# [4.0] Enable button when returning from article

Pull Request for Issue # .

### Summary of Changes

Fix disabled buttons to enabled when an element is already selected on load.

### Testing Instructions

Steps to reproduce the issue. Go to articles, select an article and then press the name of it to edit it. Press the back button to go back and you will see that the buttons change status and batch are disabled and cant be enabled even if you deselect it and then you select it again.

### Expected result

![checked-works](https://user-images.githubusercontent.com/33200407/54870203-10053080-4dac-11e9-8985-b4bd7f30b2c1.png)

### Actual result

![checked-doesnt-work](https://user-images.githubusercontent.com/33200407/54870204-15fb1180-4dac-11e9-8241-bf24f6f8163c.png)


### Documentation Changes Required
